lib/module.js

```diff
@@ -142,6 +142,8 @@ Module._findPath = function(request, paths) {
 
   // For each path
   for (var i = 0, PL = paths.length; i < PL; i++) {
+    // don't search further if path doesnt exist
+    if(paths[i] && internalModuleStat(paths[i]) <= 0) continue;
```

Comment:
There should at least be a space here between `if` and `(`. The comment should be modified slightly to read:

`// Don't search further if path doesn't exist`